plugins/inputs/bond/bond.go

```diff
@@ -118,10 +155,100 @@ func (bond *Bond) gatherBondPart(bondName string, rawFile string, acc telegraf.A
 	return fmt.Errorf("Couldn't find status info for '%s' ", bondName)
 }
 
+func (bond *Bond) readSysFiles(bondDir string) (sysFiles, error) {
+	/*
+		Files we may need
+		bonding/mode
+		bonding/slaves
+		bonding/ad_num_ports
+	*/
+	var err error
```

Comment:
You don't need to define the error as it is declared below. Please remove this line.